chapter2/tests/t2/chapter2-t2-gen-kustomization.sh

```diff
@@ -0,0 +1,53 @@
+#
+# generate_repository -- takes a repository name and generates a repository
+#
+# usage: generate_repository <NAME_REPOSITORY>
+#
+generate_repository() { 
+	local name_repository=$1
+
+# kustomize repository
+cat << EOF >> ~/github_repos/devops-camp-2023/kustomization.yaml
+  - name: $name_repository
+    namespace: argo-cd
+    options:
+      labels:
+        argocd.argoproj.io/secret-type: repository
+    literals:
+      - name=$name_repository
+      - url=git@github.com:KrawczowaKris/$name_repository.git
+      - type=git
+      - project=default
+    files:
+      - sshPrivateKey=./ssh-keys/$name_repository-deploy-key.pem
+EOF
+}
+
+# checking the introduction of arguments
+if [ $# -eq 0 ]
+	then echo "Please enter arguments"
```

Comment:
Please always make clear error messages for the end user

```suggestion
	then echo "No arguments passed. Please add repositories names to proceed."
```

Reply:
done